eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -162,8 +167,12 @@ public void onError(String errorMessage) {
                 })
                 .buildAndInit();
 
-        if(!lock.await(10000, TimeUnit.MILLISECONDS)) {
-            throw new RuntimeException("Request for RAC did not complete within timeout");
+        try {
+            if (!lock.await(10000, TimeUnit.MILLISECONDS)) {
+                throw new InterruptedException("Request for RAC did not complete within timeout");
+            }
+        } catch (InterruptedException e) {
```

Comment:
Re-attempting my clean up of catching and handling `InterruptedException` here rather than putting it in the method signature. 🧹